addons/account/models/account_payment.py

```diff
@@ -419,6 +485,16 @@ def _compute_partner_id(self):
             else:
                 pay.partner_id = pay.partner_id
 
+    @api.depends('journal_id', 'payment_type')
+    def _compute_outstanding_account_id(self):
+        for pay in self:
+            if pay.payment_type == 'inbound':
+                pay.outstanding_account_id = pay.payment_method_id.payment_account_id or pay.journal_id.payment_debit_account_id or pay.journal_id.default_account_id
+            elif pay.payment_type == 'outbound':
+                pay.outstanding_account_id = pay.payment_method_id.payment_account_id or pay.journal_id.payment_credit_account_id or pay.journal_id.default_account_id
```

Comment:
`payment_debit_account_id` and `payment_credit_account_id` no longer exist on `account.journal`

Reply:
Why not `pay.outstanding_account_id = pay.payment_method_id.account_id or pay.company_id.<default_account_in_the_company_but_I_dont_remember_its_name>`